pkg/controller/controller.go

```diff
@@ -108,11 +108,10 @@ func (c *controller) Run(workers int, stopCh <-chan struct{}) error {
 	var wg sync.WaitGroup
 	for i := 0; i < workers; i++ {
 		wg.Add(1)
-		// TODO (@munnerz): make time.Second duration configurable
-		go wait.Until(func() {
+		go func() {
 			defer wg.Done()
 			c.worker(ctx)
-		}, time.Second, stopCh)
+		}()
```

Comment:
Could we use an errgroup here too? And this is another case where I think the logic should be that all or none of the workers must be running. ...but then looking at the `worker` code, they never return any errors, only the sync function returns errors and those are already logged because they shouldn't cause the worker to exit....so maybe this is ok afterall. Maybe these things need explaining in a comment above this bit of code, if you agree.

Reply:
errgroup is basically WaitGroup + error handling, so because these controllers do only log errors, it should probably not be used here.